src/modalities/registry/components.py

```diff
@@ -106,7 +118,41 @@ class ComponentEntity:
     ),
     ComponentEntity("model", "checkpointed", ModelFactory.get_checkpointed_model, CheckpointedModelConfig),
     ComponentEntity("model", "fsdp_wrapped", ModelFactory.get_fsdp_wrapped_model, FSDPWrappedModelConfig),
+    ComponentEntity(
+        "model", "weight_initialized", ModelFactory.get_weight_initalized_model, WeightInitializedModelConfig
+    ),
     ComponentEntity("model", "coca", CoCa, CoCaConfig),
+    # weight initializers
+    ComponentEntity(
+        "weight_initialization",
+        "composed",
+        HighLevelWeightInitializationFactory.get_composed_weight_init,
+        ComposedWeightInitializationConfig,
```

Comment:
I think it is a bit suboptimal to have the naming `LowLevel` vs. `HighLevel`, but then also `Composed`. Maybe we could somehow use only `LowLevel` vs. `HighLevel` OR e.g. `Single` vs. `Composed`.

Reply:
yeah I was also unsatisfied with the naming. This should improve it: 
https://github.com/Modalities/modalities/pull/168/commits/0e29447002713b84d5bf6c0057b33b964e96cd82